src/components/sign-in/sign-in.component.jsx

```diff
@@ -1,62 +1,64 @@
 import { signInWithGoogle } from '../../firebase/firebase.utils';
 
 class SignIn extends React.Component {
-    constructor(props) {
-        super(props);
+  constructor(props) {
+    super(props);
 
-        this.state = {
-            email: '',
-            password: '',
-        };
-    }
+    this.state = {
+      email: '',
+      password: '',
+    };
+  }
 
-    handleSubmit = (event) => {
-        event.preventDefault();
+  handleSubmit = (event) => {
+    event.preventDefault();
 
-        this.setState({ email: '', password: '' });
-    };
+    this.setState({ email: '', password: '' });
+  };
 
-    handleChange = (event) => {
-        const { value, name } = event.target;
-        this.setState({ [name]: value });
-    };
+  handleChange = (event) => {
+    const { value, name } = event.target;
+    this.setState({ [name]: value });
+  };
+
+  render() {
+    return (
+      <div className="sign-in">
+        <h2>I already have an account</h2>
+        <span>Sign in with your email and password</span>
+
+        <form onSubmit={this.handleSubmit}>
+          <FormInput
+            name="email"
+            type="email"
+            value={this.state.email}
+            handleChange={this.handleChange}
+            required
+            label="Email"
+          />
 
-    render() {
-        return (
-            <div className="sign-in">
-                <h2>I already have an account</h2>
-                <span>Sign in with your email and password</span>
-
-                <form onSubmit={this.handleSubmit}>
-
-                    <FormInput
-                        name="email"
-                        type="email"
-                        value={this.state.email}
-                        handleChange={this.handleChange}
-                        required
-                        label='Email'
-                    />
-
-                    <FormInput
-                        name="password"
-                        type="password"
-                        value={this.state.password}
-                        handleChange={this.handleChange}
-                        required
-                        label='Password'
-                    />
-                    <div className='buttons'>
-                    <CustomButton type="submit" > SignIn </CustomButton>
-                    <CustomButton onClick={signInWithGoogle} isGoogleSignIn > {' '} SignIn with google {' '} </CustomButton>
-                    </div>
-                    </form>
-            </div>
-        );
-    }
+          <FormInput
+            name="password"
+            type="password"
+            value={this.state.password}
+            handleChange={this.handleChange}
+            required
+            label="Password"
+          />
+          <div className="buttons">
+            <CustomButton type="submit"> SignIn </CustomButton>
+            <CustomButton onClick={signInWithGoogle} isGoogleSignIn>
+              {' '}
```

Comment:
Consider removing `{' '}`, these usually happen when using reformatting.
There are many instances of it, try to get them all.

Reply:
Ok.